lib/main.dart

```diff
@@ -19,7 +19,7 @@ class App extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return BlocProvider(
-      create: (_) => AuthBloc(sl.get<SecureStorage>())..add(AppStarted()),
+      create: (_) => AuthenticationCubit(sl.get<SecureStorage>()),
```

Comment:
I prefer calling `appStarted` being called here instead of the call being in the constructor body, but a matter of taste